substratest/fl_interface.py

```diff
@@ -252,7 +252,7 @@ def composites_to_aggregate(model_keys):
     def aggregate_to_predict(model_key):
         return [
             InputRef(
-                identifier=InputIdentifiers.models,
+                identifier=InputIdentifiers.model,
```

Comment:
The substra fl_interface might need the same modification

Reply:
It's in the companion PR ;)

Reply:
Yes yes I know, and this specific modification isn't on it

Reply:
because there is no fl_interface in substra_fl + the test tuple inherit from composite test tuples and takes `local` and `shared` as in put for models